add-ons/bp-search-block/block.json

```diff
@@ -0,0 +1,57 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "bp/search-form",
+	"title": "Community Search",
```

Comment:
Missing `i18n`

Reply:
According to the documentation, this is automatically done, see https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#internationalization Moreover as it's a JSON file, I'm unsure i18n functions would behave as expected. I've tested with the fr_FR language, and I saw these props were included into the generated pot file by WP CLI but were not translated. That's why I'm overriding these props within the main block file (index.js).